### PR TITLE
Create a common module to post/get/delete object based on local file

### PR DESCRIPTION
**Pull request by flei2000**
Add module for file <==> object operation
Rewrite cases with the new module

### CODE REVIEW

File: ecstest/file_obj_ops.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        If the fn is not specified, generate it.
+        Return the file name.
+        '''
+        if pathfn == None:
+            pathfn = 'tmpfile.' + str(random.randint(0, 10000))
+
+        with open(pathfn, 'w') as f:
+            with open('/dev/urandom') as sf:
+                while size > 0:
+                    readsize = min(size, 4096)
```
Comment (MingatRubicon) [inline ecstest/file_obj_ops.py]: why 4k? not very efficient.

Comment (flei2000) [inline ecstest/file_obj_ops.py]: 4K is the default page size. What's your suggestion?

File: ecstest/file_obj_ops.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        '''
+        if pathfn == None:
+            pathfn = 'tmpfile.' + str(random.randint(0, 10000))
+
+        with open(pathfn, 'w') as f:
+            buff = None
+            while size > 0:
+                readsize = min(size, 1024*1024)
+                if buff == None:
+                    with open('/dev/urandom') as sf:
```
Comment (MingatRubicon) [inline ecstest/file_obj_ops.py]: no need to open close this file each time.

File: ecstest/file_obj_ops.py [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                size -= readsize
+
+        return pathfn
+
+    def __init__(self, conn, bucket_name):
+        self.conn = conn
+        self.bucket_name = bucket_name
+        self.bucket = conn.get_bucket(bucket_name)
+
+    def post(self, pathfn, objname = None):
```
Comment (MingatRubicon) [inline ecstest/file_obj_ops.py]: let us change all the place where we use "object name". there is no "object name" but "object key". key-value storage has object key, and object value.

File: ecstest/file_obj_ops.py (hunk 2 above)
Comment (flei2000) [inline ecstest/file_obj_ops.py]: It's opened/closed only the first time when buff == None.

Comment (MingatRubicon) [inline ecstest/file_obj_ops.py]: you are right. i need some morning coffee

**Pull request merged**